### ISSUE TITLE
score = torch.where(mask[i].unsqueeze(1), next_score, score)

### ISSUE THREAD

**Issue by mudongla** [yangjianxin1/LEBERT-NER-Chinese#12]
UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. 在训练MSRA数据集时碰到这个问题，而且训练时前面几个f1分数都是0，但是后面那些又正常了，这是什么原因

**Comment by mudongla** [comment]
另外F1得分不是通过训练整个数据集得出，这样F1分数是不是虚高了
